CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Notable changes
 
+* 5.2.0 - Considerable changes to make the Webpack 5 based build actually work.
```

Comment:
I'd be minded to make this a major version release given the required changes.

Reply:
I would also say that except I'd just designated the major version to track the Webpack version, with minor versions for any significant changes within that, and the patch version for compatible fixes. So a consumer could specify `~5.2.0` to get patch updates.

Reply:
Makes sense. I don't feel that strongly.